submarine-sdk/pysubmarine/submarine/experiment/api/experiment_client.py

```diff
@@ -38,6 +39,21 @@ def __init__(self, host):
         api_client = ApiClient(configuration=configuration)
         self.experiment_api = ExperimentApi(api_client=api_client)
 
+        """
+        For user login
+        :return: Passed or not
+        """
+        l = ldap.initialize(‘ldap://172.16.1.163:389‘)
+        l.protocol_version = 3
+        l.set_option(ldap.OPT_REFERRALS, 0)
+        l.simple_bind_s(user_dn, password)
+        cn = user_dn.split(‘,‘)[0].split(‘=‘)
+        base_dn = ‘dc=testad,dc=com‘
```

Comment:
`dc=testad,dc=com`, Test data can only be written in test cases.

Reply:
Well , the "testad" in here not really means a test data, because the user_dn usually be the form like this : 
dn=Eros,dc=testad,dc=com
so we need to modify it to get what we really need , which in the front case is "dn=Eros".